lib/Tests/BluetoothEngineTests/BluetoothEngineTests.swift

```diff
@@ -71,7 +71,7 @@ struct BluetoothEngineTests {
             client.onEnable = { }
             client.onConnect = { PeripheralEvent(.connect, $0) }
             await state.setSystemState(.poweredOn)
-            await state.putPeripheral(fake)
+            await state.putPeripheral(fake, replace: true)
```

Comment:
I know we're trying to hurry to get to EUT, but would probably be worth adding tests for this new argument.

Reply:
Oh yeah your're right, can do.